examples/package.json

```diff
@@ -17,6 +17,7 @@
     "fluxible-addons-react": "^0.2.0",
     "fluxible-plugin-fetchr": "^0.3.3",
     "fluxible-router": "^0.4.0",
+    "history": "^1.17.0",
     "react": "0.14.x",
     "react-dom": "0.14.x",
     "react-router": "^1.0.0-rc3",
```

Comment:
Update to `^1.0.3` here. I'm seeing weird issues with this where it's installing `1.0.2` and throwing peer invalid errors on installation.